domain/dagconfig/genesis.go

```diff
@@ -101,10 +102,10 @@ var devnetGenesisBlock = externalapi.DomainBlock{
 		[]*externalapi.DomainHash{},
 		devnetGenesisMerkleRoot,
 		&externalapi.DomainHash{},
-		&externalapi.DomainHash{},
+		externalapi.NewDomainHashFromByteArray((*[32]byte)(&muhash.EmptyMuHashHash)),
```

Comment:
It could be nice if Hash would have `.ToArray()` or something to save the `(*[32]byte)` ugliness